services/user/Chainmail/service/src/expanded_table.rs

```diff
@@ -321,8 +321,8 @@ mod service {
         type PrimaryKey = ();
         const SECONDARY_KEYS: u8 = 0u8;
         const DB: psibase::DbId = psibase::DbId::Service;
-        fn get_primary_key(&self) -> Self::PrimaryKey {
-            self.pk()
+        fn get_primary_key(&self) -> RawKey {
+            self.pk().to_key()
```

Comment:
- This is incorrect. It's from a non-working version of the macro.
- Why is this file checked in?